internal/schema/database.go

```diff
@@ -139,7 +139,39 @@ func (d *DSNDatabaseConnector) ToProto() protoreflect.ProtoMessage {
 }
 
 func (d *DSNDatabaseConnector) schemaChildren() []Node { return nil }
-func (d *DSNDatabaseConnector) schemaSymbol()          {}
+
+//protobuf:2
+type AWSIAMAuthDatabaseConnector struct {
+	Pos Position `parser:"" protobuf:"1,optional"`
+
+	Username string `parser:"" protobuf:"2"`
+	Endpoint string `parser:"" protobuf:"3"`
+	Database string `parser:"" protobuf:"4"`
```

Comment:
We don't want this to be parseable?

Reply:
Hmm, that is just cargo culting from the existing runtime, and it seems to work. Not totally sure what this does. Should these be parseable?